kms/kms.rb

```diff
@@ -174,18 +174,90 @@
       Cloudkms::CryptoKey.new(purpose: "ENCRYPT_DECRYPT")
   )
 
-  puts "Created version #{crypto_key_version.name} for key " +
-       "#{crypto_key} in key ring #{key_ring_id}"
+  version = crypto_key_version.name.split("/").last
```

Comment:
I vote revert if that's cool with you

While `name.split("/").last` is super duper common and some people will get what you're doing right away...

 1. It's complexity that I don't think is very useful for the sample
 2. It's confusing to me.  When I see `name.split("/").last` I think *"huh... um.  what?  so, I get that this is splitting on slashes and getting the last bit but... why?  What's the rest of the string?"*

This is one of those times that I value *simplicity* of sample over strict *correctness*

😄